<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="YY160371" failures="1" tests="2" name="Library" time="6.909" errors="0" timestamp="2020-08-14T15:05:08 IST">
  <testcase classname="libraryService.LibraryTest" name="LibraryPOST" time="4.904"/>
  <testcase classname="libraryService.LibraryTest" name="LibraryDelete" time="0.209">
    <failure type="java.lang.AssertionError" message="expected [book is successfully deleted] but found [books is successfully deleted]">
      <![CDATA[java.lang.AssertionError: expected [book is successfully deleted] but found [books is successfully deleted]
at libraryService.LibraryTest.LibraryDelete(LibraryTest.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- LibraryDelete -->
  <testcase classname="libraryService.LibraryTest" name="LibraryGET" time="0.0">
    <ignored/>
  </testcase> <!-- LibraryGET -->
</testsuite> <!-- Library -->
